rdbms/src/main/java/org/spine3/server/storage/jdbc/JdbcStorageFactory.java

```diff
@@ -101,25 +116,61 @@ public EventStorage createEventStorage() {
                 .newInstance(dataSource, entityStorage, false, getProjectionStorageQueryFactory(dataSource, projectionClass));
     }
 
+    /**
+     * Returns {@link AggregateStorageQueryFactory}.
+     * Override this method to use custom {@link AggregateStorageQueryFactory} extensions.
+     *
+     * @param dataSource        {@link DataSource} on which corresponding {@link JdbcAggregateStorage} is based
+     * @param aggregateClass    class of {@link Aggregate} stored in corresponding {@link JdbcAggregateStorage}
```

Comment:
You say that we store classes of aggregates. `class of aggregates which are stored` is better.